sdrbase/util/aprs.cpp

```diff
@@ -16,10 +16,25 @@
 ///////////////////////////////////////////////////////////////////////////////////
 
 #include <QRegExp>
+#include <QRegularExpression>
```

Comment:
QT5 Docs stated to use QRegularExpression over QRegExp

> Note: In Qt 5, the new QRegularExpression class provides a Perl compatible implementation of regular expressions and is recommended in place of QRegExp.

https://doc.qt.io/qt-5/qregexp.html